src/util.rs

```diff
@@ -0,0 +1,36 @@
+use std::{
```

Comment:
Ah, missed that before: can you add the license identifier please?

Reply:
Oh, missed that. Fixed.